linux.md

```diff
@@ -28,11 +28,23 @@ Make sure that you're running the 2nd-newest OS version or the newest version -
    corepack enable
    corepack prepare pnpm@latest --activate
    pnpm setup
+   source ~/`[[ $SHELL == *"zsh" ]] && echo '.zshrc' || echo '.bashrc'`
    ```
 
    This uses Corepack to install `pnpm`, and configures `pnpm`'s global bin directory.<br><br>
 
-4. Copy each line in the following text, paste it in the terminal and hit return.<br><br>
+4. Copy the following text, paste it in the terminal and hit return.<br><br>
+   ```bash
+   pnpm add --global @upleveled/preflight
+   ```
+   This will prepare a program that we will use in the course. If you encounter the following error: 
+   ```bash
+   ERR_PNPM_NO_GLOBAL_BIN_DIR  Unable to find the global bin directory
+
+   Run "pnpm setup" to create it automatically, or set the global-bin-dir setting, or the PNPM_HOME env variable. The global bin directory should be in the PATH.
+   ```
+   This indicates that the `pnpm` installation was incomplete- check the last part of the previous step to see how to finish the installation.<br><br>
+5. Copy each line in the following text, paste it in the terminal and hit return.<br><br>
```

Comment:
we can merge this into the previous step, to indicate that the previous step is not "done" yet:

````

   This uses Corepack to install `pnpm`, and configures `pnpm`'s global bin directory.<br><br>

   Install `@upleveled/preflight`, a program we will use in the course, to verify that the previous commands were successful: copy the following text, paste it in the terminal and hit return.<br><br>

   ```bash
   pnpm add --global @upleveled/preflight
   ```

   Verify that your output looks similar to the "Successful" output below. If it's more like the "Failing" output, repeat the pnpm installation commands at the start of this step.

   <table>
      <tr>
         <th>Successful</th>
         <th>Failing</th>
      </tr>
      <tr>
         <td valign="top">

   ```bash
    WARN  6 deprecated subdependencies found:
   @oclif/screen@3.0.8, glob@6.0.4, glob@7.1.6,
   osenv@0.1.5, rimraf@2.4.5, rimraf@3.0.2
   Packages: +21 -32
   +++++++++++++++++--------------------------
   Progress: resolved 773, reused 772,
   downloaded 1, added 21, done
   
   /Users/k/Library/pnpm/global/5:
   + @upleveled/preflight 7.0.8
   
   Done in 3.3s
   ```

   </td>
   <td valign="top">

   ```bash
   ERR_PNPM_NO_GLOBAL_BIN_DIR  Unable to find the global
   bin directory
   
   Run "pnpm setup" to create it automatically, or set the
   global-bin-dir setting, or the PNPM_HOME env variable.
   The global bin directory should be in the PATH.
   ```

   </td></tr></table>

5. Copy each line in the following text, paste it in the terminal and hit return.<br><br>
````

the numbering, starting with 5 at the end of the code block, needs to be updated to reflect the removed step (5 becomes 4, 6 becomes 5, etc)

apply this to all of the guides

this can be the styling for all of the steps in future, so that we have built-in verification of all steps

rendered looks like this:

![Screenshot 2024-08-13 at 14 46 35](https://github.com/user-attachments/assets/81961e6a-6fd1-47e4-bc47-f4e12570bd63)

Reply:
Added the new section for Windows, macOS and Linux in this commit
[ecb57fe](https://github.com/upleveled/system-setup/pull/91/commits/ecb57fe2e42067fd1de662f9002f6fcaf212d11c)

Reply:
the first line is not wrapped the same here, which leads to a scroll bar on a 13" screen:

![Screenshot 2024-08-22 at 12 49 43](https://github.com/user-attachments/assets/1458ecdb-0c48-49be-a0ea-1b34eb914e15)

Reply:
done in d295f336fbe786aae30ba1358e97b15c8c00e0df